extractors/database/sql_db.md

```diff
@@ -37,6 +37,13 @@ use [incremental loading](/storage/tables/#incremental-loading) when possible.
 The MySQL database server also supports encrypting the whole database communication using SSL Certificates. See the
 [official guide](http://dev.mysql.com/doc/refman/5.7/en/creating-ssl-files-using-openssl.html) for instructions on setting it up.
 
+### MS SQL Server Advanced Mode
+The SQL Server export uses the [BCP utility](https://docs.microsoft.com/en-us/sql/tools/bcp-utility?view=sql-server-2017) to export data.
+For this reason, if you are writing advanced mode queries you should properly escape quotes in text columns
+For example: `SELECT char(34) + REPLACE([my_varchar_column], char(34), char(34) + char(34)) + char(34) FROM [my_table]`
```

Comment:
please add unescaped version and then escaped version for clarity, also link microsoft docs describing what needs to be escaped and how

Reply:
there are no microsoft docs for what needs to be escaped :-), but I can find the stackoverflow entry.

Reply:
ok, now that i read the slack discussion, i understand that it works completelely different then i thought :)

so is this correct?:
"For this reason, if you are writing advanced mode queries you have to quote the values of text columns - so that the selected value is `"some text"` instead of `some text`. This can by done e.g. `SELECT char(34) + [my_text_column] + char(34)`
when the extracted text itself may contain quotes, you need to escape them (replace `"` with `""`)
 `SELECT char(34) + REPLACE([my_varchar_column], char(34), char(34) + char(34)) + char(34) FROM [my_table]`

Reply:
Yes, but I think we should remove the `[]` because that is confusing as MS uses `[]` also for identifier enclosure.

Reply:
will update it now

Reply:
actually, yes, you are right!

Reply:
ok, I updated it, thanks @odinuv !

Reply:
cool, just please fix "(ie." (and any other typos i made :)